### PR TITLE
Us31 drag and drop

### PR DESCRIPTION
**Pull request by majluni**

### CODE REVIEW

File: src/gui/dragAndDrop.js [hunk 1]
```diff
@@ -1,7 +1,49 @@
-function pickUpNode() {
-    return "TEST";
+/**
+ * Handles processing of drag and drop events for the mathematical Nodes
+ * @type {number}
```
Comment (zlmonroe) [inline src/gui/dragAndDrop.js]: What is this? Dangling docstring?

Comment (majluni) [reply src/gui/dragAndDrop.js]: I have no idea.. haven't seen that before. I'll fix it when I get home from dinner

Comment (majluni) [reply src/gui/dragAndDrop.js]: Should be gone now

**Review by JonathanBush — APPROVED** [review]
Tested and working. Might want to consider positioning dropped nodes at the position they are dropped rather than the cursor position.

**Pull request merged**